docs/src/paradox/05-internals/design/api-v2/gravsearch.md

```diff
@@ -174,16 +174,10 @@ PREFIX knora-api: <http://api.knora.org/ontology/knora-api/simple/v2#>
     }
 ```
 
-The prequery's SELECT clause is built using the member variables defined in `AbstractPrequeryGenerator`.
-State of member variables after transformation of the input query into the prequery:
-
-- `mainResourceVariable`: `QueryVariable(page)`
-- `dependentResourceVariables`: `Set(QueryVariable(book))`
-- `dependentResourceVariablesGroupConcat`: `Set(QueryVariable(book__Concat))`
-- `valueObjectVariables`: `Set(QueryVariable(book__LinkValue), QueryVariable(seqnum))`: `?book` represents the dependent resource and `?book__LinkValue` the link value connecting `?page` and `?book`.
-- `valueObjectVariablesGroupConcat`: `Set(QueryVariable(seqnum__Concat), QueryVariable(book__LinkValue__Concat))`
-
-The resulting SELECT clause of the prequery looks as follows:
+The prequery's SELECT clause is built by
+`NonTriplestoreSpecificGravsearchToPrequeryTransformer.getSelectColumns`,
+based on the variables used in the input query's `CONSTRUCT` clause.
+The resulting SELECT clause looks as follows:
 
 ```sparql
 SELECT DISTINCT
```

Comment:
Yes. This is explained in the paragraph starting on line 211:

> Each `GROUP_CONCAT` checks whether the concatenated variable is bound in each result in the group; if a variable is unbound, we concatenate an empty string. This is necessary because, in Apache Jena (and perhaps other triplestores), "If `GROUP_CONCAT` has an unbound value in the list of values to concat, the overall result is 'error'" (see [this Jena issue](https://issues.apache.org/jira/browse/JENA-1856)).